setup.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
```

Comment:
Can you please make the `setup.py` file executable? Also, please call `python2` instead of python. Some environments (such as my system) use Python 3 by default.